File: westcoast/src/main/java/frc/commands/ballhandling/IntakeOut.java
package frc.commands.ballhandling;

import edu.wpi.first.wpilibj2.command.CommandBase;
import frc.subsystems.BallHandlingSubsystem;

public class IntakeOut extends CommandBase {
  private final BallHandlingSubsystem ball;

  public IntakeOut(
    BallHandlingSubsystem ballHandler
  ) {
    this.ball = ballHandler;
    addRequirements(ballHandler);
  }

  @Override
  public void initialize() {
  }

  @Override
  public void execute() {
    ball.runintakeMotor(-0.4);
  }

  @Override
  public void end(boolean interrupted) {
    ball.stopMotors();
  }

  @Override
  public boolean isFinished() {
    return false;
  }
}

File: westcoast/src/main/java/frc/commands/ballhandling/IntakeOut.kt
package frc.commands.ballhandling

import edu.wpi.first.wpilibj2.command.CommandBase
import frc.subsystems.BallHandlingSubsystem

class IntakeOut(
    private val ball: BallHandlingSubsystem
) : CommandBase() {
  override fun initialize() {}
  override fun execute() {
    ball.runintakeMotor(-0.4)
  }

  override fun end(interrupted: Boolean) {
    ball.stopMotors()
  }

  override fun isFinished(): Boolean {
    return false
  }

  init {
    addRequirements(ball)
  }
}